lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -3298,6 +3327,63 @@ class ByteCodeBufferReader
         return current;
     }
 
+    const byte * ReadPropertyIdOfFormals(const byte * current, FunctionBody * function)
+    {
+#ifdef BYTE_CODE_MAGIC_CONSTANTS
+        int constant = 0;
+        current = ReadInt32(current, &constant);
+        Assert(constant == magicStartOfPropIdsOfFormals);
+#endif
+
+        uint32 count = 0;
+        current = ReadUInt32(current, &count);
+
+        if (count == 0)
+        {
+            return current;
+        }
+
+        int32 extraSlotCount = 0;
+        current = ReadInt32(current, &extraSlotCount);
+
+        PropertyIdArray * propIds = function->AllocatePropertyIdArrayForFormals((extraSlotCount + count) * sizeof(PropertyId),count);
+        propIds->count = count;
+        
+
+        bool hadDuplicates;
+        current = ReadBool(current, &hadDuplicates);
+        propIds->hadDuplicates = hadDuplicates;
+
+        bool has__proto__;
+        current = ReadBool(current, &has__proto__);
+        propIds->has__proto__ = has__proto__;
+
+        bool hasNonSimpleParams;
+        current = ReadBool(current, &hasNonSimpleParams);
+        propIds->hasNonSimpleParams = hasNonSimpleParams;
+
+        int id;
```

Comment:
Done.

---
In reply to: [68136835](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r68136835) [](ancestors = 68136835)